src/nvim/ex_getln.c

```diff
@@ -5957,6 +5957,27 @@ char_u *get_cmdline_str(void)
   return vim_strnsave(p->cmdbuff, (size_t)p->cmdlen);
 }
 
+// Set the current command line to a given string.
+// Only works when the command line is being edited.
```

Comment:
This comments should be in the API function, also this makes me think what happens to the cursor when you set the cmdline?

Reply:
> This comments should be in the API function

👍 

>  also this makes me think what happens to the cursor when you set the cmdline?

The cursor just goes to the end of the cmdline's text like if you had just inputted the text.